addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -377,7 +377,7 @@
             <xpath expr="//div[hasclass('o_hr_leave_column')]" position="attributes">
                 <attribute name="class" remove="col_left col-md-6" separator=" "/>
             </xpath>
-            <xpath expr="//sheet/widget[@name='web_ribbon']" position="after">
+            <xpath expr="//div[hasclass('o_hr_leave_content')]" position="before">
```

Comment:
actually this stuff will lead to a positionning offset because we don't really want the ribbons to be at the same place for the contextual

actually if we do that we'll get something like this 
![Screenshot_20240625_151423](https://github.com/odoo/odoo/assets/63317845/2f35cbe6-63d8-42cf-afea-4bd5adacc7aa)